doc/sphinx/language/core/definitions.rst

```diff
@@ -90,8 +90,9 @@ Section :ref:`typing-rules`.
    computation on :n:`@term`.
 
    These commands also support the :attr:`universes(polymorphic)`,
-   :attr:`program` (see :ref:`program_definition`),
-   :attr:`canonical` and :attr:`using` attributes.
+   :attr:`program` (see :ref:`program_definition`), :attr:`canonical`,
+   :attr:`typing(universes)`, :attr:`typing(guarded)`, and
```

Comment:
Given that `Definition`, contrary to `Theorem`, does not support the `with` syntax, it doesn't seem justified to me that this command supports the `typing(guarded)` attribute.